Backpack/FlightLeg/Classes/BPKFlightLeg.swift

```diff
@@ -0,0 +1,242 @@
+            containerStackView.trailingAnchor.constraint(equalTo: trailingAnchor)
+        ])
+    }
+    
+    private func setupCarrierLogo() -> UIView {
+        let verticalStackView = UIView()
+        verticalStackView.translatesAutoresizingMaskIntoConstraints = false
+        
+        carrierLogoIcon.iconName = carrierLogo
+        leadingStackView.addArrangedSubview(carrierLogoIcon)
+        verticalStackView.addSubview(leadingStackView)
+        
+        NSLayoutConstraint.activate([
+            leadingStackView.topAnchor.constraint(equalTo: verticalStackView.topAnchor, constant: BPKSpacingSm),
+            leadingStackView.bottomAnchor.constraint(equalTo: verticalStackView.bottomAnchor),
+            leadingStackView.leadingAnchor.constraint(equalTo: verticalStackView.leadingAnchor),
+            leadingStackView.trailingAnchor.constraint(equalTo: verticalStackView.trailingAnchor)
+        ])
+        return verticalStackView
+    }
+    
+    private func setupFlightStackView() -> UIView {
+        let departureArrivalTimeAttributes: [NSAttributedString.Key: Any] =
+            [.font: BPKFont.makeFont(fontStyle: .textHeading5)]
+        let departureArrivalTimeString = NSMutableAttributedString(
+            string: departureArrivalTime, attributes: departureArrivalTimeAttributes)
+        
+        if nextDayArrival != nil {
+            let nextDayArrivalAttributes: [NSAttributedString.Key: Any] =
+            [.font: BPKFont.makeFont(fontStyle: .textCaption), .baselineOffset: 4]
+            let nextDayArrivalString = NSMutableAttributedString(
+                string: nextDayArrival ?? "", attributes: nextDayArrivalAttributes)
+            departureArrivalTimeString.append(nextDayArrivalString)
+        }
+        departureArrivalTimeLabel.attributedText = departureArrivalTimeString
+        middleStackView.addArrangedSubview(departureArrivalTimeLabel)
+        
+        flightDescriptionLabel.attributedText = flightDescription
+        middleStackView.addArrangedSubview(flightDescriptionLabel)
+        
+        if operatedBy != nil {
+            operatedByLabel.text = operatedBy
+            middleStackView.addArrangedSubview(operatedByLabel)
+            
+        }
+        
+        if warning != nil {
+            warningLabel.text = warning
+            middleStackView.addArrangedSubview(warningLabel)
+        }
+        
+        return middleStackView
+    }
+    
+    public static func airportHighlightAttributes() -> [NSAttributedString.Key: Any] {
+        return [
+            .backgroundColor: BPKColor.statusDangerFillColor,
+            .font: BPKFont.makeFont(fontStyle: .textCaption),
+            .foregroundColor: BPKColor.textOnLightColor
+        ]
+    }
+    
+    public static func descriptionDefaultAttributes() -> [NSAttributedString.Key: Any] {
+        return [.font: BPKFont.makeFont(fontStyle: .textCaption)]
+    }
+    
+    private func setupDurationStackView() -> UIView {
```

Comment:
same as above, if it's setting up, there's no need to return it.

I would keep the setup as is maybe and just have it not return anything.